ddpui/management/commands/migrate_org_queue.py

```
@@ -208,18 +233,46 @@ def perform_migration(
+            )
+
+            # Create update schema with the new nested structure
+            update_data = QueueConfigUpdateSchema()
+            setattr(update_data, queue_type, new_queue_details)
+
+            # Apply the update
+            org.update_queue_config(update_data)
+
+            self.stdout.write(f"  ✓ Updated database config for {org.slug}")
+
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)